secret-provider/src/main/scala/io/lenses/connect/secrets/providers/VaultSecretProvider.scala

```diff
@@ -55,126 +62,85 @@ class VaultSecretProvider() extends ConfigProvider with VaultHelper {
 
   // lookup secrets at a path
   override def get(path: String): ConfigData = {
-    val (expiry, data) = cache.get(path) match {
-      case Some((expiresAt, data)) =>
-        // we have all the keys and are before the expiry
-        val now = OffsetDateTime.now()
-
-        if (expiresAt.getOrElse(now.plusSeconds(1)).isAfter(now)) {
-          logger.info("Fetching secrets from cache")
-          (expiresAt, data)
-        } else {
-          // missing some or expired so reload
-          getSecretsAndExpiry(getSecrets(path))
-        }
-
-      case None =>
-        getSecretsAndExpiry(getSecrets(path))
-    }
-
-    expiry.foreach(exp => logger.info(s"Min expiry for TTL set to [${exp.toString}]"))
-    cache += (path -> (expiry, data))
-    data
+    logger.info(" -> VaultSecretProvider.get(path: {})", path)
```

Comment:
Why is one log entry with `->` and another one with `<-`

Is this get method called a lot of times? If so, it will generate a lot of logs. The entries logged read more like debug level than info. What's the benefit, in your opinion, for having logged at INFO level

Reply:
> Why is one log entry with `->` and another one with `<-`

It was easy for me to debug/recognise/locate in the logs.
Method ingress (`->`) / Method egress (`<-`)

> Is this get method called a lot of times? If so, it will generate a lot of logs. The entries logged read more like debug level than info. What's the benefit, in your opinion, for having logged at INFO level

It will be called according to the TTL.  I didn't consider that it would be called excessively.  I'm happy to drop it down to `debug` as reducing log noise is always a good think.